python/feathub/feature_views/feature_view.py

```diff
@@ -56,15 +62,16 @@ def __init__(
             # Uses table's keys as features' keys if features' keys are not specified.
             for feature in [f for f in self.get_resolved_features() if f.keys is None]:
                 feature.keys = keys
+
+            if timestamp_field is None:
+                timestamp_field = cast(TableDescriptor, self.source).timestamp_field
```

Comment:
What happens when this feature view is un-resolved and the source table descriptor is a string? Will it raise an error or simply assign None to the timestamp_field?

Can you add a unit test if there is a bug here?

Reply:
It will simply assign None to the `timestamp_field`. I added a test to ensure the behavior.